src/example/incompressible_navier_stokes/incompressible_navier_stokes.jl

```diff
@@ -0,0 +1,304 @@
+module incompressible_navier_stokes #hide
+println("Running incompressible Navier-Stokes example...") #hide
+
+# # Incompressible Navier-Stokes (FEM) - flow around a cylinder
+# In this example, the laminar flow around a cylinder is simulated by solving the incompressible Navier-Stokes equations.
+# The equations are discretized and solved using two methods, the projection method and the "mixed form" method. This tutorial is based
+# on a [DFG Benchmark](https://wwwold.mathematik.tu-dortmund.de/~featflow/en/benchmarks/cfdbenchmarking/flow/dfg_benchmark2_re100.html) and is also an 
+# example in [Ferrite.jl](https://ferrite-fem.github.io/Ferrite.jl/stable/tutorials/ns_vs_diffeq/).
+#
+# # Description of the case
+# The simulation domain consists in a channel bounded above and below by two walls and within which a cylinder is placed. The geometrical parameters of the case
+# are shown in the Figure below. 
+#
+# ![](../assets/navier_stokes_cylindre.png)
+#
+# The flow around the cylinder is governed by the incompressible Navier-Stokes equations:
+# ```math
+#   \partial_t u + u \cdot \nabla u = - \nabla p + \nu \Delta u
+# ```
+# ```math
+#   \nabla \cdot u = 0
+# ```
+# 
+# To discretize the equations $\mathbb{P}_2$-$\mathbb{P}_1$ elements are used (Taylor-Hood). 
+# 
+# No-slip boundary conditions are applied on all walls. The inlet is given by an imposed parabolic velocity profile which
+# is ramped up in time:
+# ```math
+#   u(t,x,y) = u_{in}(t) \times \left( 4  y (0.41-y)/0.41^2 , 0 \right)
+# ```
+# where $u_{in}(t) = \textrm{clamp}(t,0.0,1.5)$. 
+# 
+# At the outlet, $p=0$ is imposed when using the projection method and a "do-nothing" ($-pn + \nu \nabla u \cdot n = 0$) condition is applied when using the "mixed form" method.
+#
+# # Code
+# Load the necessary packages
+const dir = string(@__DIR__, "/") # bcube/example dir
+using Bcube
+using LinearAlgebra
+using WriteVTK
+using StaticArrays
+
+# Function space (here we shall use Taylor-Hood P2-P1 elements) and quadrature degree.
+const fspace = :Lagrange
+const degree_u = 2
+const degquad = 2 * degree_u + 1
+const degree_p = 1
+
+# Input and output paths
+const outputpath = joinpath(dir, "../../../myout/navier_stokes/")
```

Comment:
even better would be to declare `const outputpath = joinpath(dir, "..", "..", "..", "myout," "navier_stokes/")
Comment`